src/utils/parser.ts

```diff
@@ -39,3 +40,41 @@ export function nodeToString(node: Node, code: string) {
   // Copy the string value directly from the input code
   return code.slice(start, end).trim();
 }
+
+/**
+ * Finds all css delcarations
+ * @param root PostCss AST Root
+ */
+function getCssProps(root: Root): Declaration[] {
+  const { nodes } = root;
+
+  if (!nodes) {
+    return [];
+  }
+
+  function iterate(nodes: ChildNode[], declarations: Declaration[] = []) {
+    nodes.forEach((item) => {
+      if (item.type === "decl") {
+        declarations.push(item);
+      } else if (item.type === "rule" && item?.nodes) {
+        iterate(item.nodes, declarations);
+      }
+    });
+
+    return declarations;
+  }
+
+  return iterate(nodes);
+}
+/**
+ * Parses css into PostCss AST and returns array of declarations
```

Comment:
```suggestion
 * Parses css into PostCss AST
```

That's already given by the TS return type, no need to duplicate it :wink: